src/Dapr.Actors.AspNetCore/WebHostBuilderExtensions.cs

```diff
@@ -47,6 +52,8 @@ public static IWebHostBuilder UseActors(this IWebHostBuilder hostBuilder, Action
                 services.AddRouting();
                 services.AddHealthChecks();
                 services.AddSingleton<IStartupFilter>(new DaprActorSetupFilter());
+
+                services.AddSingleton<ActorRuntime>(runtime);
```

Comment:
Isn't adding a singleton same as a static instance?

Reply:
No, it's not the same because it's only a singleton within the context of the DI container. You can't (from a static context) reach out and grab a singleton from DI - which solves all of the maintainability problems. 

The problem that this avoids is when multiple code paths want to configure something like the runtime - they have to do so by being passed an instance or an instance of a container. A true static means that you can't really audit or predict what code accesses what (it's a true global variable).

Reply:
Thanks!